package.json

```diff
@@ -3,10 +3,20 @@
   "description": "initials for names",
   "main": "initials.js",
   "scripts": {
-    "test": "standard initials.js test/initials-test.js && npm run -s test:node | tap-spec",
+    "start": "copy && beefy ./index.js --cwd=./demo",
+    "copy": "browserify lib/initials.js -s initials -o demo/initials.js",
```

Comment:
I woudl rename "copy" to "build:demo", as it's not a copy task, but a build on its own.